far/plugapi.cpp

```diff
@@ -1957,8 +1957,7 @@ intptr_t WINAPI apiEditor(const wchar_t* FileName, const wchar_t* Title, intptr_
 					/* $ 15.05.2002 SKV
 					  Зафиксируем вход и выход в/из модального редактора.
 					  */
-					if (-1 == editorExitCode)
-						Global->WindowManager->ExecuteModal(Editor);
+					Global->WindowManager->ExecuteModal(Editor);
```

Comment:
According to 384e51ec4f6e8eaa038dc286fe7b388f398fee42, the intention was to fix a crash, so I'm not quite comfortable with just reverting it like this, and tbh have neither time nor inspiration to investigate whether it's safe to do so.

Looks like in this case the exit code is `XC_OPEN_NEWINSTANCE`, maybe we can do something like this instead?
```diff
- if (-1 == editorExitCode)
+ if (any_of(editorExitCode, -1, XC_OPEN_NEWINSTANCE))
```
It should address your case and preserve the fix from 384e51ec4f6e8eaa038dc286fe7b388f398fee42, whatever it was about.
If yes, could we do that in all the affected places (filelist.cpp, findfile.cpp, usermenu.cpp) for consistency?

Reply:
@alabuzhev, thank you for your advice. Done.